data-raw/variable_description.csv

```diff
@@ -0,0 +1,40 @@
+"","register_name","raw_register_filename","years_covered","variable_name","danish_description","english_description","data_type","format","expected_values","explanation_of_values","notes_for_devs"
```

Comment:
```suggestion
"","register_name","register_abbrev","years_covered","variable_name","danish_description","english_description","data_type","format","expected_values","explanation_of_values","comments"
```

I think this should be called `register_abbrev` since that is the name of it, rather than filename, since we don't know what exactly the file name might be for any specific project.

Reply:
It should be resolved now. There's still some challenges left in how to represent expected values, Especially categorical variables. In the future we might also consider to have another set of columns for expected_values_inclusion to check if the underlying data contains the diabetes inclusion values. For now I think it's reasonable to expect it to do so if the underlying distribution looks okay. I'm also not sure how strict we're going to be with this test in practice. We'll see.